onboarding/serializers.py

```diff
@@ -283,8 +283,17 @@ class Meta:
             'page',
         )
 
+    def validate(self, data):
+        '''
+        Prevents data in Section from duplicating from backend side (when data is
+        re-send manually; problem was previously solved in frontend for the user).
+        '''
+        if Section.objects.filter(title = data['title'],
+                description = data['description'], link = data['link'],
+                owner = self.context['request'].user.company).exists():
+            raise serializers.ValidationError("Dane już istnieją w bazie - duplikat")
+        return data
```

Comment:
:+1: Good one ;-)
It is to be accepted,
but we may need to discuss in a larger group
what does it mean the section to be repeated.